dev-docs/workflows/release.md

```diff
@@ -78,58 +78,3 @@ Then wait until the node / Kubernetes upgrades are finished by periodically chec
 3.  If the release is a minor version release, bump the pre-release version in the `version.txt` file.
 4. Update the `fromVersion` in `e2e-test-release.yml` and `e2e-test-weekly.yaml` to the newly released version. To check the current values, run: `grep "fromVersion: \[.*\]" -R .github`.
 5. Reset `upgradeRequiresIAMMigration`  in `iamupgradeapply.go`.
-
```

Comment:
I would prefer to keep this documentation. While we shouldn't have to rely on those steps, it might still be handy to have them.